lib/avo/fields/base_field.rb

```diff
@@ -36,6 +36,9 @@ class BaseField
       attr_reader :nullable
       attr_reader :null_values
       attr_reader :format_using
+      attr_reader :format_display_using
+      attr_reader :format_index_using
+      attr_reader :format_show_using
```

Comment:
Let's also include `edit` `new` and `forms`

```suggestion
      attr_reader :format_display_using
      attr_reader :format_index_using
      attr_reader :format_show_using
      attr_reader :format_edit_using
      attr_reader :format_new_using
      attr_reader :format_form_using
```